sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -740,6 +740,12 @@ def __repr__(self):
 class DetectLanguageInput(LanguageInput):
     """The input document to be analyzed for detecting language.
 
+    :keyword str id: Unique, non-empty document identifier.
```

Comment:
hey @a-noyass thank you so much for doing this PR.

When adding the keyword docstrings, you can remove the `ivar` ones, so they're not duplicated. Other than that your pr looks great!

Reply:
actually @a-noyass i'm totally wrong, we should keep both. Going to merge your PR when it's green

Reply:
@iscai-msft Okay. Thank you!